src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -135,6 +135,27 @@
   <data name="Settings_FileName_Cannot_Be_A_Path" xml:space="preserve">
     <value>Parameter 'fileName' to Settings must be just a fileName and not a path</value>
   </data>
+  <data name="ShowError_ConfigInvalidOperation" xml:space="preserve">
+    <value>NuGet.Config is malformed, Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="ShowError_ConfigInvalidXml" xml:space="preserve">
+    <value>NuGet.Config is invalid XML, Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="ShowError_ConfigRootInvalid" xml:space="preserve">
+    <value>NuGet.Config is invalid, the root element is not 'configuration'. Please check NuGet.Config at '{0}'.</value>
```

Comment:
NuGet.Config does not contain the expected root element: 'configuration'. Path: '{0}'.